.evergreen/legacy_config_generator/evergreen_config_lib/tasks.py

```diff
@@ -741,7 +741,7 @@ def pre_commands(self) -> Iterable[Value]:
         [
             PostCompileTask(
                 "test-versioned-api-" + server_version,
-                tags=["versioned-api"],
+                tags=["versioned-api", f"versioned-api-{server_version}"],
```

Comment:
I believe you can specify multiple tags to refine the set of matching tasks in the variant's task list, so the `versioned-api-` prefix should be unnecessary.

```yml
# In the task definition.
tags=["versioned-api", server_version]
```

```yml
# In the variant definition.
[
    "debug-compile-nosasl-openssl",
    "debug-compile-nosasl-nossl",
    ".versioned-api .5.0",
    ".versioned-api .6.0",
],
```

Similarly for the aws tasks/variants.

Reply:
```suggestion
                tags=["versioned-api", f"{server_version}"],
```
Missed a spot.

Reply:
Oof. Thank you for the catch.